src/seedsigner/views/seed_views.py

```diff
@@ -1160,6 +1172,61 @@ def run(self):
         )
 
 
+class SeedBIP85FinalizeView(View):
+    LOAD = ButtonOption("Load Child")
+    PASSPHRASE = ButtonOption("BIP-39 Passphrase")
+    VIEW_MNEMONIC = ButtonOption("View Mnemonic")
+    
+    def __init__(self, bip85_data: dict):
+        super().__init__()
+        self.bip85_data = bip85_data
+        self.parent_seed_num = self.bip85_data["parent_seed_num"]
+        
+        self.wordlist_language_code = self.settings.get_value(SettingsConstants.SETTING__WORDLIST_LANGUAGE)
+        
+        child_seed_mnemonic = self.controller.get_seed(self.parent_seed_num).get_bip85_child_mnemonic(self.bip85_data["child_index"], self.bip85_data["num_words"]).split()
+        
+        self.controller.storage.set_pending_seed(
+            Seed(
+                mnemonic=child_seed_mnemonic,
+                wordlist_language_code=self.wordlist_language_code,
+                is_bip85_child_seed=True,
+                bip85_data=self.bip85_data,
+            )
+        )
+
+
+    def run(self):
+        button_data = [self.VIEW_MNEMONIC]
+        if self.settings.get_value(SettingsConstants.SETTING__BIP85_CHILD_SEEDS) == SettingsConstants.BIP85__ENABLED_WITH_LOADING:
+            button_data = [self.LOAD, self.PASSPHRASE, self.VIEW_MNEMONIC]
+
+        selected_menu_num = self.run_screen(
+            seed_screens.SeedBIP85FinalizeScreen,
+            child_fingerprint=self.controller.storage.get_pending_seed().get_fingerprint(network=self.settings.get_value(SettingsConstants.SETTING__NETWORK)),
+            bip85_index=self.bip85_data["child_index"],
+            button_data=button_data,
+        )
+        
+        if selected_menu_num == RET_CODE__BACK_BUTTON:
+            return Destination(SeedOptionsView, view_args={"seed_num": self.parent_seed_num}, clear_history=True)
+
+        if button_data[selected_menu_num] == self.LOAD:
+            seed_num = self.controller.storage.finalize_pending_seed()
+            return Destination(SeedOptionsView, view_args={"seed_num": seed_num}, clear_history=True)
+        
+        if button_data[selected_menu_num] == self.PASSPHRASE:
+            return Destination(SeedAddPassphraseView)
```

Comment:
The flow might feel a little disjoint, but I think I'd prefer:
* Load child
* View mnemonic

Having three button options starts to crowd the screen. And in that context, "BIP-39 passphrase" already feels a bit odd sitting above "View mnemonic".

So if we removed it, "Load child" would then just route to the existing Finalize Seed View where the bip39 passphrase option is offered. The user is already used to seeing this View so they know what it's doing.